File: Logica_de_Programacao_Python/Desafio_13/desafio_13/Desafio_13.py
vetor_nomes = []

for i in range(10):
    nome = input(f"Digite o {i + 1}º nome: ")
    vetor_nomes.append(nome)

nome_busca = input("Digite o nome que deseja buscar: ")

if nome_busca in vetor_nomes:
    print("Achei")
else:
    print("Não achei")

File: Logica_De_Programacao_Java/Desafio_13/src/Desafio_13.java
import java.util.Scanner;

public class Desafio_13 {
    public static void main(String[] args) {

        Scanner entrada = new Scanner(System.in);
        String[] nomes = new String[10];
        String nomeBusca;
        boolean encontrado = false;

        System.out.println("Digite 10 nomes:");
        for (int i = 0; i < 10; i++) {
            System.out.print("Nome " + (i + 1) + ": ");
            nomes[i] = entrada.nextLine();
        }

        System.out.print("Digite o nome para buscar: ");
        nomeBusca = entrada.nextLine();

        for (int i = 0; i < 10; i++) {
            if (nomes[i].equalsIgnoreCase(nomeBusca)) {
                encontrado = true;
                break;
            }
        }

        if (encontrado) {
            System.out.println("Achei");
        } else {
            System.out.println("Não achei");
        }

        entrada.close();
    }
}